pyproject.toml

```diff
@@ -1,11 +1,46 @@
-[build-system]
+[project]
 name = "ElectionLeaflets"
-version = "3.0-alpha"
-requires = [
-    "setuptools >= 35.0.2",
-    "setuptools_scm >= 2.0.0, <3"
+version = "0.1.0"
+description = " 📮 A Django port of the original PHP code from electionleaflets.org "
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "aws-wsgi",
+    "dc-design-system",
+    "dc-django-utils",
+    "dj-pagination==2.4.0",
+    "django-braces==1.16.0",
+    "django-extensions==3.1.5",
+    "django-filter==24.3",
+    "django-formtools==2.2",
+    "django-localflavor==4.0",
+    "django-pipeline==3.1.0",
+    "django-s3file==5.5.7",
+    "django-ses==4.3.0",
+    "django-static-jquery==2.1.4",
+    "django-storages==1.13.2",
```

Comment:
I'll be honest, I haven't really reviewed whether this is the exact right set of packages but this looks like a much more managable/correct list than we had in the old Pipfile.

However, I will flag one thing quickly.

We probably want to either install `django-storages[s3]` here, or explicitly pull in `boto3` as we will rely on `boto3` in prod for media storage. I guess because it is mocked out under test, not having it has probably not caused anything to fail locally or in CI.. or maybe something else pulls it in as a transitive package? Anyway, it feels like it should be an _explicit_ that we depend on that in prod.

Reply:
So...

We care about the zipped size of this project, and it's going to be deployed into an AWS Lambda container. This container already has `boto3` installed, and installing it in the venv only duplicates the code. 

If we ever moved to another hosting solution, we'd need to install boto, but I don't think it's worth doing in this case.

In terms of the other requirements: I've started with nothing and added them one at a time until tests and manual checking worked. I think I've got everything, but I'm sure future dev work will route out _something_ or other.

Reply:
Ah yeah I forgot you get boto3 out of the box on lambda